sql/moz-fx-data-shared-prod/search_derived/desktop_mobile_search_clients_monthly_v1/metadata.yaml

```diff
@@ -10,11 +10,18 @@ labels:
 scheduling:
   dag_name: bqetl_desktop_mobile_search_monthly
   referenced_tables: []
-  date_partition_parameter: submission_month
+  date_partition_parameter: null
+  destination_table:
+    >-
+    desktop_mobile_search_clients_monthly_v1${{
+    (execution_date +
+    macros.dateutil.relativedelta.relativedelta(months=-1,
+    day=1)).strftime("%Y%m%d")
+    }}
```

Comment:
Just to confirm this is doing the correct thing:

Let's say the task gets triggered on 2022-04-09, then this will change the partition to 20220301:

```python
from datetime import datetime
from dateutil.relativedelta import relativedelta

dt = datetime(2022, 4, 9)
delta = relativedelta(months=-1, day=1)
dt + delta

# >> datetime.datetime(2022, 3, 1, 0, 0)
```

Which is what we want, right?

Reply:
yes that is the expected behavior